src/Umbraco.Infrastructure/Serialization/JsonUdiConverter.cs

```diff
@@ -15,9 +15,19 @@ public override bool CanConvert(Type typeToConvert)
 
     /// <inheritdoc />
     public override Udi? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
-        => reader.GetString() is string value
-        ? UdiParser.Parse(value)
-        : null;
+    {
+        if (reader.GetString() is string value)
+        {
+            var validUdi = UdiParser.TryParse(value, out Udi? returnUdi);
+
+            if (validUdi)
+            {
+                return returnUdi;
+            }
+        }
+
```

Comment:
Maybe better here would be to explicitly check for "" via a `string.IsNullOrWhiteSpace` check, and then keep `UdiParser.Parse(value)`.  That would resolve the issue of handling an empty string, but would still fail obviously if there's an invalid value.  Which might be better than a silent failure as would otherwise be the case.